cmd/dinit/main.go

```diff
@@ -0,0 +1,40 @@
+package dinit
```

Comment:
Can we please rename this binary to something that makes its purpose clearer? And put a package comment explaining it in a little more detail in `doc.go`. It's hard to tell what pod(s) it's meant to live on and why it exists.